src/components/BookItem.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const BookItem = () => (
+  <div className="book-item">
+    <div className="book-details">
+      <span className="book-category">Action</span>
+      <h2>The Hunger Games</h2>
+      <span>Suzzane Collins</span>
+      <div className="links-container">
+        <ul>
+          <li>
+            <a href="@">Comments |</a>
+          </li>
+          <li>
+            <a href="##">Remove |</a>
```

Comment:
- [ ] Your **BookItem** should have a placeholder button for removing books. I would recommend you change this link to a button.

Reply:
Yeah. I get it. Thank you and I am going to implement that.